swift/ql/lib/codeql/swift/elements.qll

```diff
@@ -1,4 +1,8 @@
 // generated by codegen/codegen.py
+/**
+ * This module exports all modules providing `Element` subclasses.
+ */
```

Comment:
The style is weird:
```
//
/**
 *
 */
```
I guess we want the generated comment to be `//` so it's as [programatically] recognizable as possible.  The question is whether the following `/***/` comment in this position is accepted as comment on the file by QL-for-QL.

Reply:
It is accepted and correctly processed by `codeql generate library-doc-coverage`, which is the command used for the QLdoc coverage check.

Indeed I would prefer to keep the `//` comment as the first line, as that is used by codegen itself to keep track of generated files. It'd be possible to do otherwise, but it might be a bit more painful :slightly_smiling_face:

Reply:
If the QLdoc checker is happy, it's good enough for me.